packages/create-gatsby/src/questions/lang-ts-styles.json

```
@@ -0,0 +1,11 @@
+{
+  "gatsby-plugin-vanilla-extract": {
+    "message": "Vanilla Extract",
+    "dependencies": [
+      "gatsby-plugin-vanilla-extract",
+      "@vanilla-extract/webpack-plugin",
+      "@vanilla-extract/css",
+      "@vanilla-extract/babel-plugin"
+    ]
+  }
+}
```

```suggestion
{
  "gatsby-plugin-vanilla-extract": {
    "message": "vanilla-extract",
    "dependencies": [
      "@vanilla-extract/webpack-plugin",
      "@vanilla-extract/css",
      "@vanilla-extract/babel-plugin"
    ]
  }
}
```

Once you move it to the `styles.json` it needs to be in this shape